posts/2024-03-12/index.qmd

```diff
@@ -0,0 +1,29 @@
+---
+title: "Unveiling the Beginning of the Pandemic: The Transformative Role of Meta-Transcriptomics in Pathogen Identification"
+author: "Diego Teixeira"
+date: "2024-03-12"
+bibliography: ref.bib
+categories: [awesome science]
+---
+
+<style>
+body {
+text-align: justify}
+</style>
+
+
+The year 2020 will forever be remembered as the starting point of the COVID-19 pandemic, declared by the World Health Organization. A revolution in the speed of information generation marked this period, especially concerning the characterization of SARS-CoV-2 and the understanding of COVID-19.
+
+![Designed by [Freepik](http://www.freepik.com/)](fig1.jpg){fig-}
```

Comment:
Image name is missing, if we add nothing it will only insert Freepik credits. Example:

![image](https://github.com/Lexanomics/quarto_blog/assets/10541523/cabfe75c-df5a-46cc-af8b-e0a03c915d78)

```suggestion
![Designed by [Freepik](http://www.freepik.com/)](fig1.jpg)
```

Reply:
Done!